sdk/src/main/java/io/dapr/client/DaprClientImpl.java

```diff
@@ -1290,6 +1296,144 @@ public Mono<UnsubscribeConfigurationResponse> unsubscribeConfiguration(Unsubscri
     }
   }
 
+  /**
+   * {@inheritDoc}
+   */
+  public Mono<Void> scheduleJob(ScheduleJobRequest scheduleJobRequest) {
+    try {
+      validateScheduleJobRequest(scheduleJobRequest);
+
+      DaprProtos.Job.Builder scheduleJobRequestBuilder = DaprProtos.Job.newBuilder();
+      scheduleJobRequestBuilder.setName(scheduleJobRequest.getName());
+
+      if (scheduleJobRequest.getData() != null) {
+        scheduleJobRequestBuilder.setData(Any.newBuilder()
+            .setValue(ByteString.copyFrom(scheduleJobRequest.getData())).build());
+      }
+
+      if (scheduleJobRequest.getSchedule() != null) {
+        scheduleJobRequestBuilder.setSchedule(scheduleJobRequest.getSchedule().getExpression());
+      }
+
+      if (scheduleJobRequest.getTtl() != null) {
+        scheduleJobRequestBuilder.setTtl(scheduleJobRequest.getTtl().toString());
+      }
+
+      if (scheduleJobRequest.getRepeats() != null) {
+        scheduleJobRequestBuilder.setRepeats(scheduleJobRequest.getRepeats());
+      }
+
+      if (scheduleJobRequest.getDueTime() != null) {
+        scheduleJobRequestBuilder.setDueTime(scheduleJobRequest.getDueTime().toString());
+      }
+
+      Mono<DaprProtos.ScheduleJobResponse> scheduleJobResponseMono =
+          Mono.deferContextual(context -> this.createMono(
+                  it -> intercept(context, asyncStub)
+                      .scheduleJobAlpha1(DaprProtos.ScheduleJobRequest.newBuilder()
+                          .setJob(scheduleJobRequestBuilder.build()).build(), it)
+              )
+          );
+
+      return scheduleJobResponseMono.then();
+    } catch (Exception ex) {
+      return DaprException.wrapMono(ex);
+    }
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  public Mono<GetJobResponse> getJob(GetJobRequest getJobRequest) {
+    try {
+      validateGetJobRequest(getJobRequest);
+
+      Mono<DaprProtos.GetJobResponse> getJobResponseMono =
+          Mono.deferContextual(context -> this.createMono(
+                  it -> intercept(context, asyncStub)
+                      .getJobAlpha1(DaprProtos.GetJobRequest.newBuilder()
+                          .setName(getJobRequest.getName()).build(), it)
+              )
+          );
+
```

Comment:
Same here, we could move the GRPC response conversion to a separate method, so the main API method is clear

Reply:
Let's leave it as is